src/main/java/com/endava/pages/SuccessStoriesPage.java

```diff
@@ -12,9 +12,13 @@
 
     private static final Logger logger = LoggerFactory.getLogger(SuccessStoriesPage.class);
 
-    public static String SUCCESS_STORY_LABEL = "SUCCESS STORY | RETAIL AND CONSUMER GOODS";
+    //Not sure which case is valid, so I'll keep both until we declare which one stay
+    public static  String SUCCESS_STORY_LABEL = "SUCCESS STORIES";
+    //public static String SUCCESS_STORY_LABEL = "SUCCESS STORY | RETAIL AND CONSUMER GOODS";
 
-	public By textSSRCG = By.xpath("//figcaption/p");
+	//Xpath must be changed
```

Comment:
Please do not comment out stale references, if you find better locator, just update it without keeping old ones

Reply:
Done.